balebot/filters/template_response_filter.py

```diff
@@ -37,9 +37,8 @@ def match(self, message):
 
     def find_keywords(self, text):
         for keyword in self.keywords:
-            if keyword:
-                if text.find(keyword) != -1:
```

Comment:
not backward compatible

Reply:
It was a bug in find_keywords(). it's not backward compatible because it was not correct function